third-party/cmake/BuildLuarocks.cmake

```diff
@@ -73,13 +73,13 @@ elseif(MSVC OR MINGW)
     /LIB ${DEPS_LIB_DIR}
     /BIN ${DEPS_BIN_DIR}
     /INC ${DEPS_INSTALL_DIR}/include/luajit-2.0/
-    /P ${DEPS_INSTALL_DIR} /TREE ${DEPS_INSTALL_DIR}
+    /P ${DEPS_INSTALL_DIR}/2.4 /TREE ${DEPS_INSTALL_DIR}
     /SCRIPTS ${DEPS_BIN_DIR}
     /CMOD ${DEPS_BIN_DIR}
     ${MINGW_FLAG}
     /LUAMOD ${DEPS_BIN_DIR}/lua)
 
-  set(LUAROCKS_BINARY ${DEPS_INSTALL_DIR}/2.2/luarocks.bat)
+  set(LUAROCKS_BINARY ${DEPS_INSTALL_DIR}/2.4/luarocks.bat)
```

Comment:
Let's put the 2.4 into a variable (say after line 42):

    set(LUAROCKS_VERSION 2.4)

Reply:
Just for reference: the version is not really necessary (and confused me with https://github.com/neovim/neovim/pull/10292).
The problem with https://github.com/neovim/neovim/pull/5699 appears to have been something different likely.